File: src/Data/ExternalEggSupplier.java
package Data;

import java.util.ArrayList;

public class ExternalEggSupplier {
	public ArrayList<String> externalEggSupplier = new ArrayList<>();

	public ExternalEggSupplier() {

	}


	public String toString() {
		StringBuilder builder = new StringBuilder();
		for (String value : externalEggSupplier) {
			builder.append(value).append(" ");
		}

		return "External Egg Suppliers:\n " + builder.toString();
	}
}

File: src/data/ExternalEggSupplier.kt
package data

import java.util.*

class ExternalEggSupplier {
    var externalEggSupplier = ArrayList<String>()


    override fun toString(): String {
        val builder = StringBuilder()
        for (value in externalEggSupplier) {
            builder.append(value).append(" ")
        }

        return "External Egg Suppliers:\n $builder"
    }
}

